language-sql.md

```diff
@@ -5,3 +5,10 @@
 ### Minimum Requirements
 
 MySQL keywords in uppercase.
+
+### Database table naming convention
+
+Prefix application name to table name. E.g.: `app1_users`, `app2_log_meta`.
```

Comment:
? Do we have a short list of the applications and the conventional prefixes: like `juwai-admin` => `ja`, etc.?